src/bosh-director-core/spec/bosh/director/core/templates/rendered_job_instance_spec.rb

```diff
@@ -3,6 +3,7 @@
 require 'bosh/director/core/templates/rendered_job_template'
 require 'bosh/director/core/templates/rendered_file_template'
 require 'bosh/director/agent_client'
+require 'securerandom'
```

Comment:
It seems that this is not necessary. We had `NameError: uninitialized constant Bosh::Director::Core::Templates::SecureRandom` error on the machine without this require. Just checked on another machine and everything worked well. So, problem with local config.